Firmware/ultralcd.cpp

```diff
@@ -186,6 +186,7 @@ enum class testScreen : uint_least8_t
 
 enum class TestError : uint_least8_t
 {
+    noerror,
```

Comment:
sketch/ultralcd.cpp:7578:9: warning: enumeration value 'noerror' not handled in switch [-Wswitch]